docs/faq.md

```diff
@@ -0,0 +1,67 @@
+# FAQ
+
+### What is HORNET?
+
+HORNET is a community driven IOTA fullnode. It is written in Go which makes it lightweight and fast.
+
+---
+
+### Does HORNET run on the Mainnet?
+
+Yes, HORNET gets actively tested within the IOTA Mainnet.
```

Comment:
Actively tested doesn't sound very trustful for the only IOTA node implementation in the mainnet :)